Tasks/HelmDeployV0/src/helmcommands/helminstall.ts

```diff
@@ -81,4 +83,11 @@ export function addArguments(helmCli: helmcli): void {
         var chartPath = tl.getInput("chartPath", true);
         helmCli.addArgument("\"" + helmutil.resolvePath(chartPath) + "\"");
     }
+
+    if (version) {
+        if(semver.valid(version))
+            helmCli.addArgument("--version ".concat(version));
+        else
+            tl.debug("The given version is not valid. Running the helm install command with latest version");
```

Comment:
Add this to console log as well. The user should know why we're not honoring their version

Reply:
Ok